src/tools/merge_query.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE merged_table AS SELECT tb1.invoice_id,
```

Comment:
this should be 
`CREATE TEMP TABLE merged_table AS`
and then format your sql code
```
CREATE TEMP TABLE merged_table AS
SELECT xxxxx,
              xxxxx,
              xxxxx,
FROM table_1 AS tb1
INNER JOIN table_2 AS tb2 ON tb1.invoice_id = tb2.invoice_id
INNER JOIN table_3 AS tb3 ON tb2.invoice_id = tb3.invoice_id;
```